bzt/modules/pbench.py

```diff
@@ -143,7 +143,7 @@ def __init__(self, executor, base_logger):
         self.port = 80
         self._target = {"scheme": None, "netloc": None}
 
-    def generate_config(self, scenario, load, hostaliases):
+    def generate_config(self, scenario, load, hostaliases=()):
```

Comment:
Why leave this option? This PR removes hostaliases support completely. It's unlikely that anyone depends on this method.